### PR TITLE
add Header

### PR DESCRIPTION
**Pull request by andrianovv**

### CODE REVIEW

File: webpack/loaders/css-loaders.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ module.exports = [
     loader: 'sass-resources-loader',
     options: {
       resources: [
-        path.resolve(__dirname, '../../src/styles/main.scss')
+        path.resolve(__dirname, '../../src/styles/main.scss'),
+        path.resolve(__dirname, '../../src/styles/utils/mixins.scss'),
```
Comment (dosandk) [inline webpack/loaders/css-loaders.js]: please remove it! You must uncomment @import "utils/mixins"; in src/styles/main.scss file

File: src/styles/utils/mixins.scss [hunk 2]
```diff
@@ -18,3 +18,20 @@
     }
   }
 }
+
+@mixin set-size($width: 0px, $height: 0px) {
```
Comment (dosandk) [inline src/styles/utils/mixins.scss:22]: Nice mixins, but I can't understand why they need :)
@andrianovv Let's discuss it!

File: src/components/header/index.jsx [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+        className={styles['left-cell']}
+      >
+        <div className={styles.logo}>
+          <IconLogo />
+        </div>
       </div>
       <div className={styles['right-cell']}>
-        <div className={styles.info}>?</div>
+        <div className={styles.info}>
+          <IconQuestionCircleOutline color={'#fff'} size={19} />
```
Comment (dosandk) [inline src/components/header/index.jsx:23]: It's not good practice to pass "#fff" color via props.
We should store all colors in some config file.
I'm still thinking about it

File: src/components/header/header.scss [hunk 4]
```diff
@@ -21,19 +16,21 @@
...
+    @include set-flex(row, center, center);
     height: 100%;
+  }
+
+
+  .logo {
     width: 58px;
-    display: flex;
-    align-items: center;
+    border-right: 1px solid rgba(#cbddf2, 0.2);
```
Comment (dosandk) [inline src/components/header/header.scss]: All color must be stored in some variable!

File: src/components/header/header.scss [hunk 5]
```diff
@@ -1,16 +1,11 @@
 .header {
-    outline: 1px solid white;
-  width: 100%;
-  height: 48px;
-  background-color: #131f29;
   display: flex;
-
+  @include set-size-bg(100%, 48px, #131f29);
```
Comment (dosandk) [inline src/components/header/header.scss]: All colors must be stored in some variable!

File: src/styles/utils/mixins.scss (hunk 2 above)
Comment (andrianovv) [reply src/styles/utils/mixins.scss:22]: mixin decreases line in css and groups some css properties

**Pull request merged**